docs/getting_started.md

```diff
@@ -1,238 +1,204 @@
 
-If the desire is to adopt this template for your project and to use it with your machine learning code, it is recommended to go through this guide as it is first. This ensures everything is working on your environment. After the sample is working, follow the [bootstrap instructions](../bootstrap/README.md) to convert the ***diabetes_regression*** sample into your project starting point.
+If you already have an Azure DevOps organization, create a new project using the guide at [Create a project in Azure DevOps and TFS](https://docs.microsoft.com/en-us/azure/devops/organizations/projects/create-project?view=azure-devops).
 
-## Create a Variable Group for your Pipeline
-
-We make use of a variable group inside Azure DevOps to store variables and their
-values that we want to make available across multiple pipelines or pipeline stages. You can either
-store the values directly in [Azure DevOps](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=designer#create-a-variable-group)
-or connect to an Azure Key Vault in your subscription. Please refer to the
-documentation [here](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=designer#create-a-variable-group) to
-learn more about how to create a variable group and
-[link](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=designer#use-a-variable-group) it to your pipeline.
-Click on **Library** in the **Pipelines** section as indicated below:
-
-![library_variable groups](./images/library_variable_groups.png)
-
-Create a variable group named **``devopsforai-aml-vg``**. The YAML pipeline definitions in this repository refer to this variable group by name.
+## Get the code
 
-The variable group should contain the following required variables:
-
-| Variable Name            | Suggested Value          |
-| ------------------------ | ------------------------ |
-| BASE_NAME                | [unique base name]       |
-| LOCATION                 | centralus                |
-| RESOURCE_GROUP           | mlops-RG                 |
-| WORKSPACE_NAME           | mlops-AML-WS             |
-| AZURE_RM_SVC_CONNECTION  | azure-resource-connection|
-| WORKSPACE_SVC_CONNECTION | aml-workspace-connection |
-| ACI_DEPLOYMENT_NAME      | diabetes-aci             |
-
-**Note:**
+If you intend to contribute back to this repository, fork the project. Otherwise use the [code template](https://github.com/microsoft/MLOpsPython/generate), which copies the entire code base to your own GitHub location with the git commit history restarted. You can use the resulting repository for this guide and for your own experimentation.
```

Comment:
We found that giving simpler, "one way to do it" instructions increased completion/success rate. Since forks are so cheap on GitHub, most (all?) Azure DevOps docs unconditionally recommend forking.

Reply:
I agree we should only give one way to do it. IMO, people that want to contribute back will figure out they need to fork, so I'll remove that bit for now.

Code template vs. forking is out of scope of this PR, but an interesting discussion. One advantage of forking would be to easily get updates, especially since you don't get a choice to base the template on a specific release version. With #219 merged we'll have even more separation of the infrastructure code from the project code, which allows for easier downstream updates without conflicts. @dtzar, any thoughts?

Reply:
@vtbassmatt I generally agree.  However this IS a `code template` and is intended for someone to use for this primary purpose and are equally as "cheap" and quick to do.  Also, if they want to use this project for their own code, a fork will send them down a bad path where they'll have to start over with a code template vs easy transition from getting started.

Reply:
Good point! I was mostly referring to "give them one way to do it". (GitHub has template repos now - not sure if you've seen that feature.)

Reply:
This is a GitHub template repository - is there something I'm missing?

Reply:
Maybe using 'repository template' is better since that's the official GitHub terminology

Reply:
Also I wonder if it would help if we state it like this instead "which effectively forks the entire repository plus resets the git commit history."